lib/Wikia/src/Service/User/Preferences/PreferenceModule.php

```diff
@@ -10,68 +11,28 @@
 use Wikia\Persistence\User\Preferences\PreferencePersistenceSwaggerService;
 
 class PreferenceModule implements Module {
-	const SWAGGER_SERVICE_RAMP_USAGE = 0;
+	const PREFERENCE_CACHE_VERSION = 1;
```

Comment:
From a `$wg...` variable? Yes, but I don't think we should add to that noise, since nothing else will use that value.